kernel-util/src/main/java/org/sakaiproject/content/util/ZipContentUtil.java

```diff
@@ -121,6 +128,18 @@ public void compressFolder(Reference reference) {
 					}
 					newResourceId += ZIP_EXTENSION;
 					newResourceName += ZIP_EXTENSION;
+
+					ContentCollectionEdit currentEdit;
+					if(reference.getId().split(Entity.SEPARATOR).length>3) {
```

Comment:
So this changes it so that the displayname for the created zipfile is the displayname of the folder with a .zip extension?
And this change affects all zipfiles created, rather than just ones in the dropbox?